site/operational-guide.markdown

```diff
@@ -0,0 +1,384 @@
+number of expected peers (the 'initial peer count') in order to avoid
+small independent groups of peers from electing different leaders
+under conditions of partition.
+
+Example configurations are given in the section on deployment
+scenarios.
+
+## Electors and Observers
+
+When using consensus-based configuration then by default every peer
+included in the network is a candidate for leadership. This does not
+always make sense (for example when adding relatively short lived
+peers to an existing cluster in response to a scale-out event) and so
+an option is provided to start a peer as an _observer_. Such peers
+will not put themselves forward as leaders, but will abide by the
+results of an election amongst elector-peers.
+
+Example configurations are given in the section on deployment
+scenarios.
+
+## Persistence
+
+If you relaunch weave with _exactly_ the same parameters as the
+previous invocation (as will happen if it is restarted automatically
+by Docker), certain information is remembered:
+
+* Runtime modifications to the target peer list enacted via the
+  command line
+* The division of the IP allocation range amongst peers
+* Allocation of addresses to containers
+
+The persistence of this information is managed transparently inside
+the router container but can be destroyed explicitly if necessary.
+
+# Deployment Scenarios
+
+This section discusses various deployment scenarios, documenting the
+best-practice operational guidance for each use case.
+
+## Manual/Incremental
+
+This scenario is intended for evaluation, not production use. The
+resulting weave network will survive host reboots without the use of a
+systemd unit as long as Docker is configured to start on boot.
+
+> Author's Note: reboot assertion assumes #2029 router/proxy restart
+> policy PR is merged
+
+### Bootstrapping
+
+On initial peer:
+
+    weave launch
+    weave consense
+
+> Author's Note: the consense step here is not strictly necessary, but
+> I include it so that a uniform pattern of launch + consense is
+> ingrained as it _is_ important when extending the network
+
+### Adding a Peer
+
+On new peer:
+
+    weave launch <existing peer list>
+    weave consense
+
+The consense step is required to ensure that the new peer has joined
+to the existing network; you _must_ wait for this to complete
+successfully before moving on to add further new peers.
+
+> Author's Note: the point of serialising on consense after each
+> additional peer is to prevent the problem whereby in this scenario
+>
+>     host1$ weave launch             # quorum = 1
+>     host2$ weave launch host1       # quorum = 2
+>     host3$ weave launch host1 host2 # quorum = 2
+>
+> host1 and host2/host3 can form independent cliques if host1 is
+> partitioned from host2 and host3. An alternative (assuming #1721 is
+> fixed) would be to mandate `weave connect` on existing peers as each
+> new peer is added.
+
+Then optionally on each existing peer:
```

Comment:
This part only seems to be necessary if you don't follow the rule given above, of specifying the whole existing list of peers to each new peer.  So I would remove it.